ksql-common/src/main/java/io/confluent/ksql/util/SchemaUtil.java

```diff
@@ -280,43 +281,57 @@ public static String getSQLTypeName(Schema schema) {
     }
   }
 
-  public static String buildAvroSchema(final Schema schema, String name) {
+  public static String buildAvroSchema(final Schema schema, String name, boolean isNullable) {
 
     org.apache.avro.SchemaBuilder.FieldAssembler fieldAssembler = org.apache.avro.SchemaBuilder
         .record(name).namespace("ksql")
         .fields();
     for (Field field : schema.fields()) {
       fieldAssembler
           .name(field.name().replace(".", "_"))
-          .type(getAvroSchemaForField(field.schema()))
+          .type(getAvroSchemaForField(field.schema(), isNullable))
           .noDefault();
     }
 
     return fieldAssembler.endRecord().toString();
   }
 
-  private static org.apache.avro.Schema getAvroSchemaForField(Schema fieldSchema) {
+  private static org.apache.avro.Schema getAvroSchemaForField(Schema fieldSchema,
```

Comment:
I'm wondering why `org.apache.avro.Schema` cannot be in import?

Reply:
It has the same as KSQL `Schema` class.